scheduler.go

```diff
@@ -317,7 +335,7 @@ func (s *Scheduler) beat() {
 		entries = append(entries, e)
 	}
 	s.logger.Debugf("Writing entries %v", entries)
-	if err := s.rdb.WriteSchedulerEntries(s.id, entries, 5*time.Second); err != nil {
+	if err := s.rdb.WriteSchedulerEntries(s.id, entries, s.heartbeatInterval); err != nil {
```

Comment:
From the original issue:

> scheduler entries are prone to expire when redis is slow to respond

Even though this just affects the inspector, there is a small chance that the SchedulerEntries may be empty because they expired before the next beat ran. I would propose it to be such that redisKey ttl > beat tick.

Reply:
Absolutely, I did not want to tackle this in the same PR, but I think we should have `ttl = s.heartbeatInterval * 2`.

I don't have strong opinions on this, what option do you prefer:
- `ttl = s.heartbeatInterval * 2`
- `ttl = s.heartbeatInterval + time.Second`
- Or a second option

Reply:
> ttl = s.heartbeatInterval * 2